### ISSUE TITLE
itts:forcedDisplay should apply to image element in image profile

### ISSUE THREAD

**Issue by skynavga** [w3c/imsc#493]

**Comment by palemieux** [comment]
`itts:forcedDisplay` already applies to `div` so I do think this is necessary.

**Comment by nigelmegitt** [comment]
@skynavga we'd like to be able to resolve this before or during Thursday's call if possible.

**Comment by skynavga** [comment]
but `div` is not `image`

**Comment by palemieux** [comment]
Applying `itts:forcedDisplay` to the parent `div` of an `image` has the effect as applying it to `image` (for Image Profile document).

**Comment by skynavga** [comment]
ok, but an author should be able to apply it to the `image` itself alone, yes?

**Comment by palemieux** [comment]
> ok, but an author should be able to apply it to the image itself alone, yes?

Not necessary in IMSC, and `itts:forcedDisplay` is an IMSC extension.

My overriding suggestion is to avoid making any change to the Image Profile unless absolutely necessary.

**Comment by nigelmegitt** [comment]
> ok, but an author should be able to apply it to the `image` itself alone, yes?

@skynavga It'd be helpful to know what's driving this request - for example are there users or practitioners who have been in touch with you to complain about this omission? If it's a point of principle alone, I don't disagree that in an ideal world the semantic makes more sense if we could apply `forcedDisplay` directly to the `image`, but I would also back @palemieux 's thinking that we have a way to achieve the same effect and that making changes to the Image Profile is something we should be super reluctant to do without a strong motive, i.e. I'd also do nothing.

**Comment by skynavga** [comment]
No, there was no external input. This came up during work on TTV support for IMSC1.1 and review of upcoming IMSC1.2 semantics w.r.t. semantic symmetry and consistency.
